srcs/exec/exec_pre_in_post.c

```diff
@@ -24,7 +24,7 @@ void	exec_argv(char **argv, char **envp, t_exec *exe, t_ast *node)
 	if (ft_strchr(argv[0], '/'))
 		exec_local(argv, envp, exe, node);
 	else if (is_builtin(argv[0], &builtin_fun_ptr))
-		exec_builtin(argv, envp, exe, &builtin_fun_ptr);
+		exec_builtin(argv, envp, exe, node, &builtin_fun_ptr);
```

Comment:
Norme ?
4 paramaters max, you need to find another solution :/

Reply:
J'ignorais qu'on s'occupais de la norme maintenant, je change ca